app/src/main/java/com/cornellappdev/android/eatery/model/MealType.java

```diff
@@ -0,0 +1,25 @@
+package com.cornellappdev.android.eatery.model;
+
+public enum MealType {
+  BREAKFAST,
+  LUNCH,
+  DINNER,
+  LITE_LUNCH,
```

Comment:
Controversial opinion, but I have been wanting to remove "lite lunch" from the codebase altogether. Cornell doesn't display that info and lite lunch causes a bunch of edge cases when we try to display the menu. Thoughts?

Reply:
Cornell does display that information. If you go to their website Northstar uses it for sure (and I think one of the west does too)